worker/src/zimfarm_worker/task/worker.py

```diff
@@ -206,19 +206,23 @@ def _get_scraper_container_size(self) -> int:
         # This should be removed once the upstream PR is accepted.
         if not self.scraper:
             return 0
-        result = cast(
-            dict[str, Any],
-            self.docker.api._result(  # pyright: ignore[reportUnknownMemberType, reportAttributeAccessIssue]
-                self.docker.api._get(  # pyright: ignore[reportAttributeAccessIssue, reportUnknownMemberType]
-                    self.docker.api._url(  # pyright: ignore[reportAttributeAccessIssue, reportUnknownMemberType]
-                        "/containers/{0}/json", self.scraper.name
+        try:
+            result = cast(
+                dict[str, Any],
+                self.docker.api._result(  # pyright: ignore[reportUnknownMemberType, reportAttributeAccessIssue]
+                    self.docker.api._get(  # pyright: ignore[reportAttributeAccessIssue, reportUnknownMemberType]
+                        self.docker.api._url(  # pyright: ignore[reportAttributeAccessIssue, reportUnknownMemberType]
+                            "/containers/{0}/json", self.scraper.name
+                        ),
+                        params={"size": True},
                     ),
-                    params={"size": True},
+                    True,
                 ),
-                True,
-            ),
-        )
-        return result.get("SizeRootFs", 0) + result.get("SizeRw", 0)
+            )
+            return result.get("SizeRootFs", 0) + result.get("SizeRw", 0)
+        except Exception:
+            logger.exception("Failed to get container disk usage")
```

Comment:
I'm a bit concerned that if exception occurs too often (it shouldn't, but ...), we will fill the task worker logs with always repeating exception stack trace. And the stack trace is not that useful.

Not sure how to handle this ; could we count the number of time the problem occurred and report it at the end of the task ? log only distinct exception stack traces ?